New: SecurityScorecard Event Collector (#44420)
demisto/content: Packs/SecurityScorecard/ReleaseNotes/1_0_17.md

> #### New: SecurityScorecard Event Collector

Reviewer: ```suggestion
# New: SecurityScorecard Event Collector
```

> New: **SecurityScorecard Event Collector** integration (Available from Cortex XSIAM 8.11.0).

Reviewer: ```suggestion
Added the **SecurityScorecard Event Collector** integration (Available from Cortex XSIAM 8.11.0) that does the following:  
```
@YaelShamai Cortex XSIAM 8.11.0 is not a valid version number (it's either 3.x for legacy or 5.x for Platform)

> 

Reviewer: ```suggestion
```

> - Collects history events from SecurityScorecard for ingestion into Cortex XSIAM.

Reviewer: ```suggestion
  - Collects history events from SecurityScorecard for ingestion into Cortex XSIAM.
```

> - Enriches each event with detailed information from the event's detail URL.

Reviewer: ```suggestion
  - Enriches each event with detailed information from the event's detail URL.
```

> - Handles API rate limiting gracefully by sending collected events and waiting for the next fetch cycle.

Reviewer: ```suggestion
  - Handles API rate limiting gracefully by sending collected events and waiting for the next fetch cycle.
```

> - Supports deduplication of events across fetch cycles.

Reviewer: ```suggestion
  - Supports deduplication of events across fetch cycles.
```
